src/main/java/com/nekomaster1000/secondchanceforge/config/SecondChanceConfig.java

```diff
@@ -27,30 +27,36 @@
 
     public SecondChanceConfig(Builder builder) {
         coyoteTimeEnabled = builder
-                .comment("Determines if the 'Coyote Time' feature is enabled\n" +
+                .comment("Determines if the 'Coyote Time' feature is enabled.\n" +
                         "'Coyote Time' allows the player to jump for a second or two after falling off of a block in order to stop themselves from falling into a pit\n" +
                         "This will not allow the player to double jump, only jump when walking off of a block")
+                .translation(SecondChanceForge.MOD_ID + ".config.tooltip.coyoteTimeEnabled")
                 .define("coyoteTimeEnabled", true);
 
         coyoteTimeTicks = builder
                 .comment("Determines how long (in ticks) the player has to jump with 'Coyote Time' after falling off a block")
+                .translation(SecondChanceForge.MOD_ID + ".config.tooltip.coyoteTimeTicks")
                 .defineInRange("coyoteTimeTicks", 10, 1, Integer.MAX_VALUE);
 
         secondChanceEnabled = builder
                 .comment("Determines if the 'Second Chance' feature is enabled\n" +
```

Comment:
I saw you added punctuation to this line in the lang file, but not here. Was this intentional?